samples/manager/manager.js

```diff
@@ -0,0 +1,1946 @@
+// Copyright 2017 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/* eslint-disable prefer-destructuring */
+
+'use strict';
+
+const fs = require('fs');
+
+// [START iot_get_client]
+const {google} = require('googleapis');
```

Comment:
Do we still need this import?

Reply:
removed, hopefully there are no side effects